content/getting-started/installation.md

```diff
@@ -21,65 +21,41 @@ does. The output should look something like this:
 
 ```bash
 $ java -version
-java version "1.8.0_11"
-Java(TM) SE Runtime Environment (build 1.8.0_11-b12)
-Java HotSpot(TM) 64-Bit Server VM (build 25.11-b03, mixed mode)
+java version "1.8.0_131"
+Java(TM) SE Runtime Environment (build 1.8.0_131-b11)
+Java HotSpot(TM) 64-Bit Server VM (build 25.131-b11, mixed mode)
 ```
 
 ```bash
 $ javac -version
-javac 1.8.0_11
+javac 1.8.0_131
 ```
 
-You need Java 8, which will show up as version 1.8. Java 7 is no longer supported in Scalatra 2.5.
+You need Java 8 (Oracle's JDK or OpenJDK), which will show up as version 1.8. Java 7 is no longer supported in Scalatra 2.5.
 
-If you don't yet have Java installed, you can find out how to install
-it for your system
-[over at the Java 8 install page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html). Make sure you're using OpenJDK or Sun's JDK.
+If you don't yet have Java installed, you can find out how to install it for your system over at
+[the Oracle's Java 8 download page](http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html)
+or [the OpenJDK installation page](http://openjdk.java.net/install/index.html).
 
-Some Linux distros pre-install `gcj`, which won't work.
+## 2. Install sbt
 
-## 2. Install giter8
+Once you're sure you've got Java installed, you will need to install the _sbt_.
 
-Once you're sure you've got Java installed, you will need to download a few
-other utilities.
+[sbt (simple build tool)](http://www.scala-sbt.org) is a build tool for Scala, Java, and more.
```

Comment:
If I understand correctly, the sbt project no longer expresses the definition of sbt as 'simple build tool'. I sometimes see 'Scala Build Tool' is used instead. Anyway, in my opinion, the `(simple build tool)` here is unnecessary.